composer.json

```diff
@@ -59,7 +60,15 @@
     "config": {
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": true,
-            "phpro/grumphp": true
+            "phpro/grumphp": true,
+            "phpstan/extension-installer": true
+        }
+    },
+    "extra": {
+        "phpstan/extension-installer": {
+            "ignore": [
+                "mglaman/phpstan-drupal"
```

Comment:
Circleci tests would fail if this is run inside this package without Drupal. Before only one rule was included the below phpstan.neon but now phpstan/extension-installer was icluding also this.